avaframe/com1DFA/com1DFA.py

```diff
@@ -1311,6 +1311,9 @@ def DFAIterate(cfg, particles, fields, dem, simHash=''):
     massEntrained = []
     massTotal = []
 
+    # setup a result fields info data frame to save max values of fields and avalanche front
+    resDF = setupResDF(resTypesLast, cfg['VISUALISATION'].getboolean('createRangeTimeDiagram'))
```

Comment:
Would you mind renaming this to something like resultsDF resuDF ... (same for the functions)?
res is very much preloaded with 'resistance in the context of com1dfa...

Reply:
renamed to resultsDF and setupResultSDF now